modules/ROOT/pages/index.adoc

```diff
@@ -23,6 +23,162 @@ Anypoint Platform for PCF uses the following services:
 
 * **Anypoint Metering Service**: provides an API that enables you to view usage metrics for Mule applications deployed to PCF. You can view usage metrics for production, pre-production, staging, and development environments.
 
+== What PCF Supports
+
+When running Mule on Pivotal Cloud Foundry (PCF) and using the cloud-hosted control plane, deploy Runtime Fabric for Mule deployments alongside your PCF environment, as this provides better integration of Anypoint Platform, support for managed patches, and new runtime versions.
+
+If you choose not to use the cloud-hosted control plane, use the native PCF tooling for deploying and managing Mule on PCF.
+
+=== What does MuleSoft provide?
+
+==== Buildpacks for Mule runtimes
+MuleSoft provides a buildpack for each of the following versions of the Mule runtime. These buildpacks enable you to deploy their applications to PCF using native tooling.
+
+***** Mule 3.9.1
+***** Mule 3.8.7
+***** Mule 3.8.5
+***** Mule 3.8.1
+==== Anypoint Platform Tile
+The Anypoint Platform Tile contains 3 service brokers, which perform the following actions:
+
+***** Manage Mule applications in PCE v1.6.1 or earlier.
+***** Manage API policies in PCE v1.6.1 or earlier.
+***** Count incoming transactions to deployed Mules.
+
+The first two service brokers are no longer compatible with Anypoint Private Cloud Edition (PCE) as of v1.7 and later. 
+The third service broker is used to count the number of incoming transactions for each Mule application or API. Coupled with a metering plugin which resides on each Mule buildpack, this service broker generates reports for auditing and licensing purposes. The service was built to support a transaction-based pricing model for Mules deployed on PCF.
+
+=== What is supported with PCF?
+
+==== On-premise standalone
+
+===== Mule Application Deployments
+In practice, customers use the native tooling (API, CLI) that PCF provides to deploy and manage their Mule applications similar to other applications they deploy and manage on PCF. 
+
+With the native tooling, customers can perform the following actions:
+
+****** Deploy applications with CLI, API, or a common build pipeline
+****** Set custom properties.
+****** Horizontally scale their Mule applications by increasing the number of replicas.
+****** Specify the version of the Mule runtime per application.
+****** Specify the amount of memory for each application replica.
+****** Cluster Mule runtimes by setting up an externalized HazelCast service broker.
+****** Configuring the application endpoint name.
+****** Start, stop, and delete Mule applications.
+****** View monitoring data and set alerting via PCF metrics.
+****** Re-deploy without downtime, with support for blue/green deployments.
+****** Automatic failover of Mule applications.
+
+===== API Policy Management 
+API policies are supported with the following additional setup:
+
```

Comment:
Rework so that you do not have a sentence that ends with a colon introducing a heading that ends with a colon.
Note how distant the "heading" below is from its narrative peer.